internal/project/model.go

```diff
@@ -26,3 +29,14 @@ func (m *Model) Key() string {
 func (m *Model) Prefix() string {
 	return common.GetBasePrefix() + Prefix
 }
+
+// FetchProjectID returns the key for storing in the registry
+func FetchProjectID(val string) string {
+	stringArr := strings.Split(val, "/")
+	arrLen := len(stringArr)
+	if arrLen < 1 {
+		log.Fatalf("Invalid ProjectID given as input: [%v]", val)
```

Comment:
Please use the logger package from `pkg` folder. Do not use `log` directly.

Reply:
Sure, updating this. We should change this similarly in different places where this is done which I took as reference.